www/source/partials/docs/_bp-planwriting-guide.html.md.erb

```diff
@@ -106,4 +106,4 @@ All plans should have a README. Items to strongly consider including:
 - Documentation on how to scale the service.
 - Instructions on how to monitor the health of the service at the application layer.
 - Can a user simply call the package as a dependency of their application?
-- How does the package integrate into their 
\ No newline at end of file
+- How does the package integrate into theirs
```

Comment:
this sentence could be reworded to something like "How does this package integrate into their application?"

Reply:
Done :)